src/benchmark_runner.cc

```diff
@@ -138,12 +142,16 @@ void RunInThread(const BenchmarkInstance* b, IterationCount iters,
 class BenchmarkRunner {
  public:
   BenchmarkRunner(const benchmark::internal::BenchmarkInstance& b_,
-                  std::vector<BenchmarkReporter::Run>* complexity_reports_)
+                  const size_t outer_repetitions_,
+                  const size_t inner_repetitions_,
```

Comment:
these can both be `int` (and actually everywhere you're using `int64_t` for repetitions can also be `int`.

Reply:
DONE and changed all int64_t to int